src/styles/_catalog.scss

```diff
@@ -9,14 +9,23 @@
     display: flex;
     justify-content: center;
     align-items: center;
+
     width: 200px;
     height: 300px;
+
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
-  }
 
-  &__card:not(:nth-child(4n)) {
-    margin-right: 48px;
+    transition: 300ms;
```

Comment:
do not add `transition` for `all` properties, it is bad performance-wise. Specify only the properties you need.